README.md

```diff
@@ -510,9 +510,46 @@ pong-service            NodePort    10.96.148.241   <none>        8443:30530/TCP
 ```
 We see we have 2 exposed ports here, we want to use 9443, this is where the example app displays a webpage.
 In the example above we can find `ping`'s external service on port 32719 and `pong`'s service on port 31017.
-If you open the browser and go to `https://<hostname>:<port>` it will display the certificate details of the other service that it internally contacted inside the Kubernetes cluster.
+These are [NodePort Services](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport),
+which means that Kubernetes will open a TCP port on each of the cluster nodes
+and forward connections to the Pods that are selected by that service.
 
-![pingpong certificate details](./images/pingpong.png)
+In order to connect, we first need to discover the IP address of the Node, using:
+
+```console
+kubectl describe nodes  | grep InternalIP
```

Comment:
style whise this differs from the lab
in the rest it is all
```
```console
$ kubectl
output
```
```

Reply:
Fixed.